en/docs/extend/sdks/spring-boot.md

```diff
@@ -160,7 +163,7 @@ Try to access any of the application pages. It will redirect to WSO2 Identity Se
 
 ### Customize the login page and use another login endpoint
 
-Even though Spring Boot generates a default login page, follow the instructions given below to customize the page as required.
+Even though Spring Boot generates a default login page for us, we can define our own customized page as required by following the instructions given below.
```

Comment:
IMO, the previous sentence is simpler and more precise. We avoid using "we", "our" etc as much as possible since we don't want it to be in a sales tone.